code/modules/admin/verbs/ahelp_template.dm

```diff
@@ -0,0 +1,238 @@
+/proc/get_html(title as text, js as text, css as text, content as text)
```

Comment:
this whole thing should be in the modular folder

Reply:
Didn't know we had one

Reply:
it's the yogstation/yogstation folder 😅